pmlc/target/intel_gen/BUILD

```diff
@@ -42,6 +42,9 @@ plaidml_cc_library(
         "@llvm-project//mlir:LLVMTransforms",
         "@llvm-project//mlir:SCFToGPUPass",
         "@llvm-project//mlir:StandardOpsTransforms",
-    ],
+    ] + select({
+        "//plaidml:gpu_target": ["//pmlc/rt/vulkan"],
+        "//conditions:default": [],
+    }),
```

Comment:
Seems like we should always have `intel_gen` target depend on the vulkan runtime.

Reply:
Thanks for your note. I think we can run keras backend tests with --//plaidml:target=intel_gen in plaidml-v1. This PR is not needed. I will close it. Thanks.